src/app/main/component/events/components/event-details/event-details.component.ts

```diff
@@ -71,7 +71,7 @@ export class EventDetailsComponent implements OnInit, OnDestroy {
   public rate: number;
 
   deleteDialogData = {
-    popupTitle: 'homepage.events.delete-title',
+    popupTitle: this.roles.ADMIN ? 'homepage.events.delete-title-admin' : 'homepage.events.delete-title',
```

Comment:
It's not important is user or admin. The message need to be the same
User story https://github.com/ita-social-projects/GreenCity/issues/3196
![image](https://user-images.githubusercontent.com/104850911/230781355-03b8f665-e6ca-4515-ba26-1f459225f8ec.png)

Reply:
ok i changed it